internal/engine/synclet_build_and_deployer.go

```diff
@@ -171,3 +172,7 @@ func (sbd *SyncletBuildAndDeployer) updateViaExec(ctx context.Context,
 
 	return nil
 }
+
+func (sbd *SyncletBuildAndDeployer) ForceUpdateViaExec(b bool) {
```

Comment:
could we just inject the UpdateMode via Wire instead of having a setter?

Reply:
Yeah, let me play around with that.